editor/find_in_files.cpp

```diff
@@ -150,19 +150,19 @@ void FindInFiles::_iterate() {
 			pop_back(folders_to_scan);
 
 			_current_dir = _current_dir.plus_file(folder_name);
+			//print_line(String("Scanning ") + _current_dir);
```

Comment:
I agree that such commented line should be removed. It's easy enough to add a print or a breakpoint when you do want the output, but leaving unused commented out debug code all over the place just makes things harder to read.

Reply:
I removed them.